src/widgets/aboutDialog.js

```diff
@@ -19,27 +19,11 @@
  */
 
 const {GObject, Gtk} = imports.gi;
+const ABOUT_RESOURCE = "/com/github/tchx84/Flatseal/widgets/aboutDialog.ui";
+function showAboutDialog(window) {
+    let builder = Gtk.Builder.new_from_resource(ABOUT_RESOURCE);
```

Comment:
Code styling nit, but these should be declared with the `const` keyword

Reply:
Thanks, its not my code, but I have no clue about gjs or js so I appreciate these style/good practices comments. Fixed.